docs/en/status-communities/understand-token-requirements-in-communities.md

```diff
@@ -0,0 +1,35 @@
+---
+id: 490
+revision: 0
+language: en
+hide:
+  - navigation
+---
+
+# Understand token requirements in communities
+
+<!--
+image
+-->
+
+In Status, there are two kinds of communities: public and private. If you wish to join a private community, you need to meet the community owner token requirements. To maintain access to a private community, hold the necessary tokens at all times.
+
+Community owners can configure token requirements for their communities. To join a community you need to meet its token requirements. For example, a community owner can set up 10 SNts as a requirement to join a community. So you need to have a minimum of 10 SNTs to join.
```

Comment:
- 'configure' -> 'set up'
- 'To join a community**,**' 
- '<...> can set up owning 10 SNTs <...>'
- 'In that case, you need to have a minimum of 10 SNTs to join that community' - I'm also not sure we need this sentence as it sort of feels redundant?